src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/DiscoverVariantsFromContigAlignmentsSAMSpark.java

```diff
@@ -66,6 +69,10 @@
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME)
     private String vcfOutputFileName;
 
+    private static boolean isImpreciseDeletion(Tuple2<PairedStrandedIntervals, EvidenceTargetLink> e) {
+        return e._1.getLeft().getInterval().getContig() == e._2().getPairedStrandedIntervals().getRight().getInterval().getContig() && e._1.getLeft().getStrand() && !e._1.getRight().getStrand();
```

Comment:
Actually moved the method to `EvidenceTargetLink` since it seems to belong there. Added some documentation to `StrandedInterval`, which mostly just refers people to the definition of strand in `BreakpointEvidence`.